### ISSUE TITLE
errors in macOS / OF master branch

### ISSUE THREAD

**Issue by moebiussurfing** [tettou771/ofxComponent#1]
hey @tettou771 , thanks for sharing.

do you think it could be easy to repair this for macOS?
it seems it's not detecting the namespace to get the methods from the addon...

```
Error:Build failed with 20 errors and 2 warnings in 11 s 678 ms
ofApp.h
Error:(9, 32) reference to 'Component' is ambiguous
Error:(9, 32) base specifier must name a class
Error:(10, 17) only virtual member functions can be marked 'override'
Error:(15, 16) only virtual member functions can be marked 'override'
Error:(11, 3) use of undeclared identifier 'setRect'
Error:(12, 3) use of undeclared identifier 'setRotation'
Error:(13, 3) use of undeclared identifier 'setScale'
Error:(17, 25) use of undeclared identifier 'getWidth'
Error:(17, 37) use of undeclared identifier 'getHeight'; did you mean 'ofGetHeight'?
Error:(20, 11) use of undeclared identifier 'getMouseX'; did you mean 'ofGetMouseX'?
Error:(20, 26) use of undeclared identifier 'getMouseX'; did you mean 'ofGetMouseX'?
Error:(20, 40) use of undeclared identifier 'getWidth'
Error:(21, 11) use of undeclared identifier 'getMouseY'; did you mean 'ofGetMouseY'?
Error:(21, 26) use of undeclared identifier 'getMouseY'; did you mean 'ofGetMouseY'?
Error:(21, 40) use of undeclared identifier 'getHeight'; did you mean 'ofGetHeight'?
Error:(23, 17) use of undeclared identifier 'getMouseX'; did you mean 'ofGetMouseX'?
Error:(23, 30) use of undeclared identifier 'getMouseY'; did you mean 'ofGetMouseY'?
Error:(26, 25) use of undeclared identifier 'getMousePos'
Error:(27, 21) use of undeclared identifier 'localToGlobalPos'
ofApp.cpp
Error:too many errors emitted, stopping now [-ferror-limit=]
main.cpp
Error:too many errors emitted, stopping now [-ferror-limit=]
```

**Comment by tettou771** [comment]
It looks conflict of naming. I didn't found this bug bedause it made with windows.
I'm gonna fix it to use it with Mac!  (but, I can't make that compatible to old versions, sorry)

ofxComponent is developing.
I'm gonna more priority grow it up than compativility.

**Comment by tettou771** [comment]
I resolve conflict of naming (5f0e553).
Component -> ofxComponentBase
ComponentManager -> ofxComponentManager
and header/cpp file name.

If you already builded the example, please update (or regenerate) example projecct with project generator.

fixed #1
